src/descriptors/power_system_structs.json

```diff
@@ -228,6 +228,12 @@
                     "comment": "Name of the component. Components of the same type (e.g., `PowerLoad`) must have unique names, but components of different types (e.g., `PowerLoad` and `ACBus`) can have the same name",
                     "data_type": "String"
                 },
+                {
+                    "null_value": "false",
```

Comment:
@jd-lara should this be true?

Reply:
I rather make the null value false to avoid that `ACBus(nothing)` gets added to a system and works.